tests/test_update.py

```diff
@@ -134,7 +131,8 @@ def putup(self, *args, with_coverage=False, **kwargs):
         if with_coverage:
             # need to pass here as list since its args to coverage.py
             args = [subarg for arg in args for subarg in arg.split()]
-            putup_path = path_join(self.venv_path, 'bin', 'putup')
```

Comment:
This won't work on windows... So I replaced it with `which`

Reply:
Nice, didn't know about `which` being available in `shutil`.

Reply:
Yes, I discovered it last week when messing around with this issue. I guess it is a wonderful Python 3 addition.

Python's stdlib always surprises me.